src/html-pipe.js

```diff
@@ -82,13 +82,19 @@ export async function htmlPipe(state, req) {
     }
 
     // ...and apply the folder mapping
-    await folderMapping(state);
+    state.timer?.update('content-fetch');
+    let contentPromise = await fetchContent(state, req, res);
```

Comment:
```suggestion
    const contentPromise = await fetchContent(state, req, res);
```

Am I'm misreading this? `contentPromise` isn't being re-defined later on.

Reply:
yes it is, on line 90